Anamnesis/Files/SceneFile.cs

```diff
@@ -6,12 +6,12 @@ namespace Anamnesis.Files;
 using Anamnesis.Actor;
 using Anamnesis.GUI.Dialogs;
 using Anamnesis.Memory;
-using Serilog;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text.Json.Serialization;
```

Comment:
Is this unneeded? Im only working in the Github Dif view so I cant see full context.